### PR TITLE
#18 - add organizations list

### LINKED ISSUE #18: Organizations list
The goal of this task is to make public view for organizations.

### PR DESCRIPTION
**Pull request by JakubZakowicz**
I have created simple views for organizations list and forms to create and edit them and to organizations profiles.

- View to organizations list
organizations_page

- View to organization creation form
create_organization

- View to organization editing form with crud to organization profiles
edit_organization

- View of form to create or edit organization profile
create_edit_profile

This should close #18

### CODE REVIEW

File: resources/views/components/form-input.blade.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div>
+    <label for="{{ $id }}" class="block font-medium text-gray-700">
+        {{ $name }}
```
Comment (kamilpiech97) [inline resources/views/components/form-input.blade.php:3]:
```suggestion
        {{ $label }}
```

File: resources/views/components/form-input.blade.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    </label>
+    <div class="mt-1">
+        <input type="{{ $type }}" name="{{ $id }}" id="{{ $id }}"
+            placeholder="{{ $name }}..." value="{{ $value }}"
```
Comment (kamilpiech97) [inline resources/views/components/form-input.blade.php:7]:
```suggestion
            placeholder="{{ $placeholder }}" value="{{ $value }}"
```

File: resources/views/components/form-input.blade.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div>
+    <label for="{{ $id }}" class="block font-medium text-gray-700">
+        {{ $name }}
+    </label>
+    <div class="mt-1">
+        <input type="{{ $type }}" name="{{ $id }}" id="{{ $id }}"
```
Comment (kamilpiech97) [inline resources/views/components/form-input.blade.php:6]:
```suggestion
        <input type="{{ $type }}" name="{{ $field }}" id="{{ $id }}"
```
I would add more readable variable for name attribute.

File: src/Models/Organization.php [hunk 4]
```diff
@@ -17,7 +17,7 @@
  * @property string $location
  * @property string $organizationType
  * @property string $foundationDate
- * @property string $numberOfEmployers
+ * @property string $numberOfEmployees
```
Comment (kamilpiech97) [inline src/Models/Organization.php:20]: I think `numberOfEmployers` should be an integer.

File: database/migrations/2022_03_09_170615_create_organizations_table.php [hunk 5]
```diff
@@ -16,7 +16,7 @@ public function up(): void
             $table->string("location");
             $table->string("organization_type");
             $table->dateTime("foundation_date");
-            $table->string("number_of_employers");
+            $table->string("number_of_employees");
```
Comment (kamilpiech97) [inline database/migrations/2022_03_09_170615_create_organizations_table.php:19]: Please create new one migration for incoming changes in `organizations` table.

Comment (kamilpiech97) [reply database/migrations/2022_03_09_170615_create_organizations_table.php:19]: You should not changing existing migrations which were created in another PR.

**Review by kamilpiech97 — CHANGES_REQUESTED** [review]

File: resources/views/meetups/create.blade.php [hunk 6]
```diff
@@ -1,58 +1,62 @@
...
+              <label for="description" class="block font-medium text-gray-700">
+                Description
+              </label>
+              <div class="mt-1">
+                <textarea rows="5" type="text" name="description" id="description" placeholder="Description..."
+                  class="shadow-sm focus:ring-indigo-500 focus:border-indigo-500 block w-full sm:text-sm border-gray-300 rounded-md">{{ old('description') }}</textarea>
+                <x-input-error for="description" />
+              </div>
+            </div>
+            <x-form-input id="date" field="date" label="Date" placeholder="Date..." type="datetime-local"
```
Comment (kamilpiech97) [inline resources/views/meetups/create.blade.php:39]:
```suggestion
            <x-form-input id="date" field="date" label="Date" placeholder="Date" type="datetime-local"
```
I would prefer delete that `...` in placeholders.

**Review by kamilpiech97 — CHANGES_REQUESTED** [review]

File: database/factories/OrganizationFactory.php [hunk 7]
```diff
@@ -20,7 +20,8 @@ public function definition(): array
             "location" => $this->faker->country(),
             "organization_type" => $this->faker->word(),
             "foundation_date" => Carbon::createFromDate(2022, 01, 01),
-            "number_of_employers" => $this->faker->randomNumber(),
+            "number_of_employees" => $this->faker->randomNumber(),
+            "logo_path" => $this->faker->url(),
```
Comment (MichalMyskow) [inline database/factories/OrganizationFactory.php:24]:
```suggestion
```
You should delete this line, we are creating `logo_path` with model.

File: src/Http/Requests/Organization/StoreOrganizationRequest.php [hunk 8]
```diff
@@ -16,8 +16,8 @@ public function rules(): array
             "location" => ["required"],
             "organization_type" => ["required"],
             "foundation_date" => ["required"],
-            "number_of_employers" => ["required"],
-            "logo" => ["image", "max:2048"],
+            "number_of_employees" => ["required"],
+            "logo" => ["required", "image", "max:2048"],
```
Comment (MichalMyskow) [inline src/Http/Requests/Organization/StoreOrganizationRequest.php:20]:
```suggestion
            "logo" => ["image", "max:2048"],
```
Logo is not required when we have default logo system.

**Review by MichalMyskow — CHANGES_REQUESTED** [review]

**Review by MichalMyskow — COMMENTED** [review]
`organization_logo.jpeg` file should be removed.

**Pull request merged**